node-src/types.ts

```diff
@@ -141,8 +141,16 @@ export interface Context {
 
   git: {
     version: string;
+    /**
+     * The creator of the build is:
+     *   - the current user in local builds
+     *   - the creator of the commit in CI builds
+     *   - 'unknown' in CI situations where we cannot read the current commit info.
+     */
+    creatorEmail: string;
```

Comment:
Within the context of the `git` object, this should be `userEmail`. Our APIs of course should still reference `creatorEmail` (or e.g. `creatorId`).

Reply:
I was using `userEmail` to refer to the `git config user.email`, where as `creatorEmail` is the slightly more complicated thing referenced in the comment above. I wanted to add the complicated thing to the `git` object because we need to pass it to various APIs along the way, so I didn't want to recalculate it each time (although I could).